README.md

```diff
@@ -35,29 +33,20 @@ cd spark-redis
 mvn clean package -DskipTests
 ```
 
-### Jedis and read-only Redis cluster slave nodes
```

Comment:
I have create https://github.com/RedisLabs/spark-redis/issues/76 to review this functionality and possibly merge into master.